job_controller/job.go

```diff
@@ -299,3 +309,14 @@ func (jc *JobController) cleanupJob(runPolicy *apiv1.RunPolicy, jobStatus apiv1.
 	jc.WorkQueue.AddRateLimited(key)
 	return nil
 }
+func getPriorityClassName(runPolicy *apiv1.RunPolicy) string {
+	priorityClassName := *runPolicy.SchedulingPolicy.PriorityClassName
+	return priorityClassName
+}
+func getTotalReplicas(replicas map[apiv1.ReplicaType]*apiv1.ReplicaSpec) int32 {
+	jobReplicas := int32(0)
+	for _, r := range replicas {
+		jobReplicas += *r.Replicas
```

Comment:
I think we should check if r.Replicas is nil here, too.

Reply:
I got it